README.asciidoc

```diff
@@ -131,6 +134,13 @@ The plugin defines the following extension properties in the `docker` closure:
 |`apiVersion`    |String    |null                     |The remote API version. For most cases this can be left null.
 |=======
 
+[WARNING]
+====
+Default value of `docker.url` property is deprecated.
+It will be replaced to `unix:///var/run/docker.sock` in upcoming releases.
+Please adjust your build scripts accordingly.
```

Comment:
@redi-foran I agree in principle but we have backwards compatibility for existing users to think of. I'm in favor of defaulting to secure by default, as that's what docker does, but not without at least a few minor releases under our belt and giving folks ample time to adjust.